test/api/test_rest.py

```diff
@@ -522,6 +522,17 @@ def test_testruns_metrics(self):
         data = self.hit('/api/testruns/%d/metrics/' % self.testrun.id)
         self.assertEqual(list, type(data['results']))
 
+    def test_testruns_status(self):
```

Comment:
Not to be too picky, but could you please add a test where you test Status' filtering?